locale/tr.po

```diff
@@ -19329,175 +18795,154 @@ msgid ""
 "                         frequencies are both ~a Hz).~%~\n"
 "                         Please select a frequency range."
 msgstr ""
+"~aBant genişliği sıfırdır (üst ve alt~%~\n"
+"                          frekansların her ikisi de ~a Hz'dir).~%~\n"
+"                          Lütfen bir frekans aralığı seçin."
 
 #: plug-ins/SpectralEditShelves.ny
-#, fuzzy
 msgid "Spectral edit shelves"
-msgstr "Spektral Seçim"
+msgstr "Spektral düzenleme rafları"
 
 #: plug-ins/StudioFadeOut.ny
-#, fuzzy
 msgid "Studio Fade Out"
-msgstr "Azalarak Çık"
+msgstr "Studyo Karartma Çıkışı"
 
 #: plug-ins/StudioFadeOut.ny plug-ins/adjustable-fade.ny
-#, fuzzy
 msgid "Applying Fade..."
-msgstr "Uygulanıyor..."
+msgstr "Solma Uygulanıyor..."
 
-#: plug-ins/StudioFadeOut.ny plug-ins/adjustable-fade.ny
-#: plug-ins/crossfadeclips.ny plug-ins/crossfadetracks.ny plug-ins/delay.ny
+#: plug-ins/StudioFadeOut.ny plug-ins/adjustable-fade.ny plug-ins/crossfadeclips.ny
+#: plug-ins/crossfadetracks.ny plug-ins/delay.ny
 #: plug-ins/eq-xml-to-txt-converter.ny plug-ins/equalabel.ny
 #: plug-ins/label-sounds.ny plug-ins/limiter.ny plug-ins/noisegate.ny
 #: plug-ins/sample-data-export.ny plug-ins/sample-data-import.ny
 #: plug-ins/spectral-delete.ny plug-ins/tremolo.ny
-#, fuzzy
 msgid "Steve Daulton"
-msgstr "&Varsayılan Ayarla"
+msgstr "Steve Daulton"
 
 #: plug-ins/StudioFadeOut.ny
-#, fuzzy, lisp-format
+#, lisp-format
 msgid "Selection too short.~%It must be more than 2 samples."
-msgstr "Seçimde en az %d örnek bulunmalı."
+msgstr "Seçim çok kısa.~% 2'den fazla örnek olmalıdır."
 
 #: plug-ins/adjustable-fade.ny
-#, fuzzy
 msgid "Adjustable Fade"
-msgstr "Ayarlanmış kaydırma"
+msgstr "Ayarlanabilir Solma"
 
 #: plug-ins/adjustable-fade.ny
-#, fuzzy
 msgid "Fade Type"
-msgstr "&Süzgeç Türü:"
+msgstr "Karartma Türü"
 
 #: plug-ins/adjustable-fade.ny
-#, fuzzy
 msgid "Fade Up"
-msgstr "Artarak Gir"
+msgstr "Karart"
 
 #: plug-ins/adjustable-fade.ny
-#, fuzzy
 msgid "Fade Down"
-msgstr "Artarak Gir"
+msgstr "Karartma Aşağı"
 
 #: plug-ins/adjustable-fade.ny
-#, fuzzy
 msgid "S-Curve Up"
-msgstr "Eğri Adı"
+msgstr "S-Eğrisi Yukarı"
 
 #: plug-ins/adjustable-fade.ny
-#, fuzzy
 msgid "S-Curve Down"
-msgstr "Aşağı Taşı"
+msgstr "S-Eğrisi Aşağı"
 
 #: plug-ins/adjustable-fade.ny
 msgid "Mid-fade Adjust (%)"
-msgstr ""
+msgstr "Orta-Solma Ayarı (%)"
 
 #: plug-ins/adjustable-fade.ny
-#, fuzzy
 msgid "Start/End as"
-msgstr "Başlangıç - Bitiş"
+msgstr "Başla/Bitir"
```

Comment:
sonunda as olduğu için başka bir ifade izliyor olsa gerek. Şunun gibi başla/bitir: daha yerinde olabilir. Programdan kontrol edilmeli.